manifests/d/DawidPieper/ELTEN/2.4.2.3/DawidPieper.ELTEN.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: DawidPieper.ELTEN
+PackageVersion: 2.4.2.3
+PackageLocale: en-US
+Publisher: Dawid Pieper
+PackageName: ELTEN
+License: Copyright © 2014-2021 Dawid Pieper

```

Comment:
A copyright is not a license; Please provide actuall license terms such as `Proprietary`, `MIT`, `GPL-2.0`, or other terms as applicable